networking/workloads-outside-cluster.md

```diff
@@ -1,49 +1,47 @@
 ---
-title: Configure workload access outside the cluster
-description: Enable and disable internet access for your workloads.
+title: Configure outgoing NAT
+description: Configure Calico Enterprise networking to perform outbound NAT for connections from pods to outside of the cluster, NATing the source IP from the pod IP to the node IP of the node hosting the pod.
 canonical_url: '/networking/workloads-outside-cluster'
 ---
 
 ### Big picture
 
-With all {{site.prodname}} networking options, you can enable and disable internet access for your workloads.
+Configure {{site.prodname}} networking to perform outbound NAT for connections from pods to outside of the cluster, NATing the source IP from the pod IP to the node IP of the node hosting the pod.
 
 ### Value
 
-You control which {{site.prodname}} IP pools are subject to outbound NAT to enable public and private network connections and allow single IP address communication.
+When to enable outgoing NAT on connections for pods, depends on your networking implementation and what you are trying to accomplish. The {{site.prodname}} NAT outbound connection option is flexible; it can be enabled, disabled, and applied to {{site.prodname}} IP pools with public IPs, private IPs, or a specific range of IP addresses. This article describes some use cases for enabling and disabling outgoing NAT.
```

Comment:
wdyt about starting the paragraph with the last sentence and removing some fluff? maybe:

> This article describes some use cases for enabling and disabling outgoing NAT. The {{site.prodname}} NAT outbound connection option is flexible; it can be enabled, disabled, and applied to {{site.prodname}} IP pools with public IPs, private IPs, or a specific range of IP addresses.

Reply:
Value statements should always lead with the value first, so let's meet in the middle and shorten it, but leave the "this article" at the end.